19.2.1 (#3)
friedcucumber/wpf-themedesigner-app: CHANGELOG.md

# Changelog

> This file lists all the notable changes to this project.

Reviewer: "all the notable changes" Why 'notable'?

## Theme Designer 19.2.1 - 2019-12-09

### New Features

> * The [Visual Tree Window](http://docs.devexpress.com/WpfThemeDesigner/401169/workspace/visual-tree-window) displays the current preview's visual element structure.

Reviewer: "displays the current" Why 'current'? Should there be some distinction between other previews?

    <img src="media/Changelog/19.2.1-VisualTree-2x.png" width="50%" alt="Theme Designer 19.1.3 - Visual Tree Window"/>

* The [Properties Window](http://docs.devexpress.com/WpfThemeDesigner/401212/workspace/properties-window) displays the selected element's properties.
    
    <img src="media/Changelog/19.2.1-Properties-2x.png" width="50%" alt="Theme Designer 19.1.3 - Properties Window"/>

* The **Ignore System DPI** tool ignores the system's scaling value and displays the preview with 100% scale. 

### Bugs Fixed

> * <a href="https://www.devexpress.com/Support/Center/Question/Details/T754663/theme-designer-the-preview-window-is-corrupted-when-the-custom-theme-name-is">T754663</a> - The preview window is corrupted when the custom theme name is Office2019Colorful.

Reviewer: "is corrupted" Explain

Author: Is corrupted = Is broken. 

That's the support ticket's header.

> * <a href="https://www.devexpress.com/Support/Center/Question/Details/T719715/theme-designer-preview-is-empty-in-certain-cases">T719715</a> - Theme Designer Preview is empty in certain cases.

Reviewer: ```suggestion
* <a href="https://www.devexpress.com/Support/Center/Question/Details/T719715/theme-designer-preview-is-empty-in-certain-cases">T719715</a> - The Theme Designer Preview is empty in certain cases.
```

Reviewer: "in certain cases" is vague. Can you be more specific?

Author: That's the support ticket's header.

### Enhancements

> * A palette color's tooltips added to the CodeView's Intelligent Code Completion popup window.

Reviewer: ```suggestion
* A palette color's tooltips are added to the CodeView's Intelligent Code Completion pop-up window.
```

    <img src="media/Changelog/19.2.1-Code-Completion-2x.png" width="50%" alt="Theme Designer 19.1.3 - Code Completion Enhancement"/>

### Changed

> * The **View in XAML** tool has been replaced with the **Inspect Element** tool. **Inspect Element** allows you to select a UI element to display its properties, XAML code, and its position in the Visual Tree.

Reviewer: ```suggestion
* The **View in XAML** tool was replaced with the **Inspect Element** tool. **Inspect Element** allows you to select a UI element and display its properties, XAML code, and position in the Visual Tree.
```

> * The **Filter Colors** tool now displays only the colors the **selected element and its children** uses. You can use the Inspect Element tool or the **Visual Tree Window** to select an element.

Reviewer: ```suggestion
* The **Filter Colors** tool now displays only the colors from the **selected element and its children**. You can use the Inspect Element tool or **Visual Tree Window** to select an element.
```

> * Theme Designer now uses the Ribbon's Simplified mode.

Reviewer: ```suggestion
* The Theme Designer now uses the Ribbon's Simplified mode.
```

    <img src="media/Changelog/19.2.1-Simplified-Ribbon-2x.png" width="50%" alt="Theme Designer 19.1.3 - Ribbon Simplified Mode"/>

> * The **Reset Layout** tool has been moved to the 'Home' ribbon tab.

Reviewer: ```suggestion
* The **Reset Layout** tool was moved to the 'Home' ribbon tab.
```

> * The **Views** ribbon tab has been converted to the drop-down button and moved to the 'Home' ribbon tab.

Reviewer: ```suggestion
* The **Views** ribbon tab was converted to a drop-down button and moved to the 'Home' ribbon tab.
```

## Theme Designer 19.1.4 - 2019-05-27

### Bug Fixes

> * <a href="https://www.devexpress.com/Support/Center/Question/Details/T743008/theme-designer-cannot-upgrade-a-theme-because-the-x-namespace-prefix-is-not-defined-in">T743008</a> - Theme Designer cannot upgrade a theme because the 'x' namespace prefix is not defined in certain files.

Reviewer: ```suggestion
* <a href="https://www.devexpress.com/Support/Center/Question/Details/T743008/theme-designer-cannot-upgrade-a-theme-because-the-x-namespace-prefix-is-not-defined-in">T743008</a> - The Theme Designer cannot upgrade a theme because the 'x' namespace prefix is not defined in certain files.
```

Reviewer: "in certain files" Can you be more specific?

## Theme Designer 19.1.3 - 2019-05-17

### Changes

> * The 'Palettes' button in the Palette ribbon tab is renamed to 'Import'.

Reviewer: Why "is renamed"? Do you mean 'was renamed'?
